versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+   ops-manager-db-svc              ClusterIP      None          <none>         27017/TCP                        14m
+   ops-manager-svc                 ClusterIP      None          <none>         8080/TCP                         13m
+   ops-manager-svc-ext             LoadBalancer   10.7.32.125   34.23.212.14   8080:31348/TCP,25999:31914/TCP   13m
+   ```
+
+8. To generate Ops Manager URL address if service is exposed as a LoadBalancer using the following command:
+ 
+   :::caution
+   The below command will only work if your service is exposed as a loadbalancer in case it is exposed as nodeport service then kindly 
+   use the URL as given below
+   
+   ```yaml
+   URL=http://<External Node IP>:<NodePort>
+   ```
+
+   where `External Node IP` of the worker node where Ops Manager is deployed & `NodePort` is the nodeport on which `ops-manager-svc-ext` 
+   is exposed.
+   :::
+
+   Example
+   ```
+   URL=http://$(kubectl -n "mongodb-operator" get svc ops-manager-svc-ext -o jsonpath='{.status.loadBalancer.ingress[0].ip}:{.spec.ports[0].port}') echo $URL
+   ```
+   
+   Example Output
+   ```
+   http://34.23.212.14:8080
+   ```
+
+9. Update the Ops Manager Kubernetes manifest to include an external IP address created by LoadBalancer in 
+   the `spec.configuration.mms.centralUrl` through `kubectl patch` using the following command:
+
+   ```
+   kubectl -n "mongodb-operator" patch om ops-manager --type=merge -p "{\"spec\":{\"configuration\":{\"mms.centralUrl\":\"${URL}\"}}}" mongodbopsmanager.mongodb.com/ops-manager patched
+   ```
+
+   :::caution
+   Wait for few minutes. The Ops Manager pod must be restarted, so wait until the `ops-manager-0` pod is in the running state again.
+   :::
+
+10. Using the username and password stored in the `om-admin-secret` (as created under the third sub-step of Step 6), log in to the Ops Manager 
+    User Interface using the address in the $URL variable.
+
+    ![mongodb](/images/version1.3.0/use-cases/configure-mongodb/ops-manager-ui.png)
+
+11. Kubernetes Operator is in the Ops Manager `ops-manager-db` organization and the `ops-manager-db` project.
+
+    ![mongodb](/images/version1.3.0/use-cases/configure-mongodb/ops-organizations.png)
+
+12. If we click on the `ops-manager-db` project, we will be redirected to the panel where we can see the database pods of the Ops 
+    Manager application. Ops Manager monitors this database.
+
+    ![mongodb](/images/version1.3.0/use-cases/configure-mongodb/ops-manager-db.png)
+
+    ![mongodb](/images/version1.3.0/use-cases/configure-mongodb/ops-deployment.png)
+
+
+### Step 7: Deploying MongoDB Across Multiple Kubernetes Clusters With MongoDBMulti
+
+:::info
+To know more, see [deploying MongoDB across multiple Kubernetes clusters with MongoDBMulti](https://www.mongodb.com/developer/products/connectors/deploying-across-multiple-kubernetes-clusters/).
+:::
+
+::note
```

Comment:
This note is not formatted correctly 

![image](https://github.com/user-attachments/assets/3a24ab5c-fec2-4377-8acd-d73e1aa2e8d9)